### PR TITLE
netex - creating netex2ntfs bin without tests

### PR DESCRIPTION
**Pull request by prhod**
This PR aims to ease the merge of the work made on PR https://github.com/CanalTP/navitia_model/pull/92 
The content of this PR : 
- creating the netex2ntfs bin
- mutualizing with gtfs the config file reading  and definition of dataset's validityperiod
There are no tests, they will be the next PR

### CODE REVIEW

File: src/read_utils.rs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    if let Some(config_path) = config_path {
+        let json_config_file = File::open(config_path)?;
+        let config: Config = serde_json::from_reader(json_config_file)?;
+        info!("Reading dataset and contributor from config: {:?}", config);
+
+        contributor = config.contributor;
+
+        use chrono::{Duration, Utc};
+        let duration = Duration::days(15);
+        let today = Utc::today();
```
Comment (TeXitoi) [inline src/read_utils.rs]: Why?

Comment (TeXitoi) [reply src/read_utils.rs]: I think you forgotten to remove the old version of this function.

Comment (prhod) [reply src/read_utils.rs]: you are right ! Using the new function implies some refacto, I've opened an issue for this point to separate issues : https://github.com/CanalTP/navitia_model/issues/98

**Comment by datanel** [comment]
@TeXitoi can we merge?

File: src/netex/mod.rs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            Some("zip") => {
+                let zip_file = fs::File::open(path)?;
+                let mut zip = zip::ZipArchive::new(zip_file)?;
+                for i in 0..zip.len() {
+                    let mut file = zip.by_index(i)?;
+                    match file.sanitized_name().extension() {
+                        None => info!(
+                            "Netex read : skipping file in ZIP : {:?}",
+                            file.sanitized_name()
+                        ),
```
Comment (TeXitoi) [inline src/netex/mod.rs]: useless branch

**Pull request merged**